curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-getters-and-setters-to-control-access-to-an-object.english.md

```diff
@@ -39,7 +39,7 @@ Getters and setters are important because they hide internal implementation deta
 
 ## Instructions
 <section id='instructions'>
-Use the <code>class</code> keyword to create a Thermostat class. The constructor accepts a Fahrenheit temperature.
+Use the <code>class</code> keyword to create a Thermostat class. The constructor accepts a Fahrenheit temperature. This constructor should convert Fahrenheit to Celsius.
 Now create a <code>getter</code> and a <code>setter</code> in the class, to obtain the temperature in Celsius.
 Remember that <code>C = 5/9 * (F - 32)</code> and <code>F = C * 9.0 / 5 + 32</code>, where <code>F</code> is the value of temperature in Fahrenheit, and <code>C</code> is the value of the same temperature in Celsius.
```

Comment:
A lot of confusion comes from users being given the formula for temperature conversion after the instruction to create a `getter`. With what you have added makes it sound like:
> Create a constructor that converts an input Fahrenheit into Celcius.
> Create a getter that converts Fahrenheit into Celcius, and
> create a setter that converts Celcius into Fahrenheit.

I would suggest a lot more of the challenge be re-worded, ensuring a decent workflow be implemented.

Reply:
@Sky020 okay, Ill make an edit to it tonight and try to change the wording around